Detectors/CTP/simulation/src/Digitizer.cxx

```diff
@@ -69,10 +69,16 @@ std::vector<CTPDigit> Digitizer::process(const gsl::span<o2::ctp::CTPInputDigit>
           break;
         }
         case o2::detectors::DetID::EMC: {
-          for (auto const& ctpinp : det2ctpinp[o2::detectors::DetID::EMC]) {
-            uint64_t mask = (inp->inputsMask).to_ullong() & detInputName2Mask[ctpinp.name];
-            if (mask) {
-              inpmaskcoll |= std::bitset<CTP_NINPUTS>(ctpinp.inputMask);
+          if (mEMCsim == 0) {
+            if (inp->inputsMask.to_ullong() & detInputName2Mask["MBA"]) {
+              inpmaskcoll |= std::bitset<CTP_NINPUTS>(CTP_NINPUTS - 1);
+            }
+          } else {
+            for (auto const& ctpinp : det2ctpinp[o2::detectors::DetID::EMC]) {
+              uint64_t mask = (inp->inputsMask).to_ullong() & detInputName2Mask[ctpinp.name];
+              if (mask) {
+                inpmaskcoll |= std::bitset<CTP_NINPUTS>(ctpinp.inputMask);
+              }
```

Comment:
Note that also with the trigger simulation we will have min. bias accept, as the min. bias was downsampled in data. So one can maybe remove the mEMCsim and treat the MBA in both cases the same setting the highest bit in the trigger inputs.

Reply:
HI @mfasDa , I am not sure if I understand this and previous comment. I understand definition of problem is:
'Suppress writing to ctpreadout all trigger classes which are related to EMC cluster with the exception if I receive EMCinput with bit 0'.
If I do not have mEMCsim how do I know that I should fully ignore TVX for EMC classes ?